### PR TITLE
COV-42: create user DB table schemas 

### PR DESCRIPTION
**Pull request by jason-gerard**
JIRA: https://soen-390-covid-tracker.atlassian.net/browse/COV-42

## Context
This pr adds the table schema for users and user roles so that the data can be stored and queried in later prs.

## Changes
- Adds table schemas for `users`, `roles`, `user_roles`
- Adds basic db init script
- Adds basic db seed script

image
image
image

### CODE REVIEW

File: server/.env.example [hunk 1]
```diff
@@ -1 +1,6 @@
 PORT=80
+PGHOST=localhost
```
Comment (domsec) [inline server/.env.example:2]: Add the PG env elements (lines 2 to 6) inside the code block of step 2 under Setup in the README

File: README.md [hunk 2]
```diff
@@ -31,7 +31,12 @@ Before you continue, ensure you have installed the following requirements:
     $ docker-compose up -d server
     $ docker-compose logs -f server
     ```
-5. (Optional) Connect to the database on `localhost:5432` in pgAdmin
+5. (Optional) Initializing and seeding the database
+    ```sh
+    $ npm run db:init
+    $ npm run db:seed
+    ```
+8. (Optional) Connect to the database on `localhost:5432` in pgAdmin
```
Comment (domsec) [inline README.md]: Should be step 6 and not 8

File: README.md [hunk 3]
```diff
@@ -72,6 +77,22 @@ Run linter and formatter with the following command in the `server` directory
 $ npm run lint:fix
 ```
 
+### Resetting the database
```
Comment (domsec) [inline README.md]: Capitalize D in database

File: server/.env.example (hunk 1 above)
Comment (jason-gerard) [reply server/.env.example:2]: I went ahead and refactored the `.env` files to a better pattern. This way unless you are building to deploy to production you dont need to touch any of the `.env` files and we can customize them for tests, development, production, etc

**Review by Andre-Ibrahim — APPROVED** [review]
looks good

**Pull request merged**